From 7b36a88440c59dbb84c9112b54852202ccbdca3c Mon Sep 17 00:00:00 2001
From: Matthieu Vincenot <mvincenot@nvidia.com>
Date: Mon, 2 Jul 2012 17:18:54 +0100
Subject: [PATCH 2/2] Enable use of modem. property by radio user
X-NVConfidentiality: public

This allow FILD triggers to reenable RIL service after modem FW
download.

bug 1009251

Reviewed-on: http://git-psac/r/332
(cherry picked from commit 09ba85c2db5da5f5a7d51f3a783ac4fe051c35f0)

Change-Id: I277ea2918c6570af03e72c7d28e67da0f1ea6078
Reviewed-on: http://git-master/r/115822
Tested-by: Sang-Hun Lee <sanlee@nvidia.com>
Reviewed-by: Prajakta Gudadhe <pgudadhe@nvidia.com>

Conflicts:

	init/property_service.c
---
 init/property_service.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/init/property_service.c b/init/property_service.c
index f58e07d..ef9f9bd 100755
--- a/init/property_service.c
+++ b/init/property_service.c
@@ -94,6 +94,7 @@ struct {
     { "persist.security.", AID_SYSTEM,   0 },
     { "persist.service.bdroid.", AID_BLUETOOTH,   0 },
     { "selinux."         , AID_SYSTEM,   0 },
+    { "modem.",            AID_RADIO,   0},
     { NULL, 0, 0 }
 };
 
-- 
1.7.9.5

